src/tzdata/TZData.jl

```diff
@@ -2,14 +2,14 @@ module TZData
 
 using LazyArtifacts
 using Printf
-using ...TimeZones: DEPS_DIR
+using ...TimeZones: TZJFile, DEPS_DIR
 
 # Note: The tz database is made up of two parts: code and data. TimeZones.jl only requires
 # the "tzdata" archive or more specifically the "tz source" files within the archive
 # (africa, australasia, ...)
 
 const TZ_SOURCE_DIR = joinpath(DEPS_DIR, "tzsource")
-const COMPILED_DIR = joinpath(DEPS_DIR, "compiled", string(VERSION))
+const COMPILED_DIR = joinpath(DEPS_DIR, "compiled", "tzjf")
```

Comment:
We should specify the tzjfile version as part of this path. Mainly this only matters when we start to use scratch spaces and have multiple tzjfile versions.

As we write the `DEFAULT_VERSION` I'll just reference that here but I can see a future iteration being more explicit about the compiled version being build and the directory it's output to.